auction-server/src/opportunity/service/get_missing_signers.rs

```diff
@@ -0,0 +1,17 @@
+use {
+    super::{
+        ChainTypeSvm,
+        Service,
+    },
+    crate::opportunity::entities,
+    solana_sdk::pubkey::Pubkey,
+};
+
+impl Service<ChainTypeSvm> {
+    pub fn get_missing_signers(&self, opportunity: entities::OpportunitySvm) -> Vec<Pubkey> {
```

Comment:
this function doesn't need &self. Maybe it's better to implement it inside OpportunitySvm?